lua/gitblame/init.lua

```diff
@@ -7,12 +7,16 @@ local M = {}
 ---@type integer
 local NAMESPACE_ID = vim.api.nvim_create_namespace("git-blame-virtual-text")
 
----@type table<string, string>
-local last_position = {}
+---@type PositionInfo
+local last_position = {
+    filepath = nil,
+    line = -1,
+    is_on_same_line = false,
+}
```

Comment:
The type here isn't exactly `table<string, string>` so I set the type it should be and also set a proper default value

![image](https://github.com/f-person/git-blame.nvim/assets/35381535/b6eb569b-a235-4f70-87fb-85cb8d4ecec1)